packages/MarkdownEditor-Core.package/MarkdownBlockTextStyler.class/instance/buildTextFrom..st

```diff
@@ -0,0 +1,6 @@
+parsing
+buildTextFrom: aCollectionOfMarkdownBlocks
```

Comment:
Here we could just return the `runs` of the text being created. Furthermore, renaming this method to something like `attributesFor: ...` would be nice.